pkg/services/live/demultiplexer.go

```diff
@@ -0,0 +1,51 @@
+package live
+
+import (
+	"context"
+
+	"github.com/grafana/grafana-live-sdk/telemetry/telegraf"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+type Demultiplexer struct {
+	telegrafConverterWide *telegraf.Converter
+	managedStreamRunner   *ManagedStreamRunner
+}
+
+func NewDemultiplexer(managedStreamRunner *ManagedStreamRunner) *Demultiplexer {
+	return &Demultiplexer{
+		telegrafConverterWide: telegraf.NewConverter(),
+		managedStreamRunner:   managedStreamRunner,
+	}
+}
+
+func (s *Demultiplexer) GetHandlerForPath(_ string) (models.ChannelHandler, error) {
+	return s, nil
+}
+
+func (s *Demultiplexer) OnSubscribe(_ context.Context, _ *models.SignedInUser, e models.SubscribeEvent) (models.SubscribeReply, backend.SubscribeStreamStatus, error) {
+	reply := models.SubscribeReply{}
+	return reply, backend.SubscribeStreamStatusOK, nil
+}
+
+func (s *Demultiplexer) OnPublish(_ context.Context, _ *models.SignedInUser, evt models.PublishEvent) (models.PublishReply, backend.PublishStreamStatus, error) {
+	stream, err := s.managedStreamRunner.GetOrCreateStream("archer")
```

Comment:
should be somethign like
```suggestion
	stream, err := s.managedStreamRunner.GetOrCreateStream("$namespace$")
```

Reply:
fixed